set.mm

```diff
@@ -127694,13 +127694,13 @@ this axiom (with the defined operation in place of ` + ` ) follows as a
   $( $j restatement 'ax-addf' of 'axaddf'; $)
 
   $( Multiplication is an operation on the complex numbers.  This deprecated
-     axiom is provided for historical compatibility but is not a bona fide
-     axiom for complex numbers (independent of set theory) since it cannot be
-     interpreted as a first-order or second-order statement (see
-     ~ https://us.metamath.org/downloads/schmidt-cnaxioms.pdf ).  It may be
-     deleted in the future and should be avoided for new theorems.  Instead,
-     the less specific ~ ax-mulcl should be used.  Note that uses of ~ ax-mulf
-     can be eliminated by using the defined operation
+     axiom is provided for historical compatibility.  However, while Metamath
+     can handle it, it cannot be interpreted as a first-order or second-order
+     statement.  We generally prefer simpler statements (see
+     ~ https://us.metamath.org/downloads/schmidt-cnaxioms.pdf ).  This
+     deprecated axiom may be deleted in the future and should be avoided for
+     new theorems.  Instead, the less specific ~ ax-mulcl should be used.  Note
```

Comment:
"Instead, the less specific ~ ax-mulcl should be used." Is this (still) true? The usage of ~ax-mulcl is also discouraged, and ~mulcl should be used instead. ~ax-mulcl is currently usded in the proof for ~mulcl only.

Reply:
Yes, ~ax-mulcl should not be used directly, it should be used only through ~mulcl. As of today, all theorems obey this rule.